### ISSUE TITLE
Update log message alerting rules

### ISSUE THREAD

**Issue by steven-sheehy** [hashgraph/hedera-mirror-node#434]
**Detailed Description**
We had a testnet issue that caused mirror node to fail file validation but we never received an alert since the log message was not in ops rules. We need to update the troubleshooting doc to:

- Remove log messages that are no longer relevant
- Update log messages that have changed
- Scan the code for any new messages and add them

Once PR is approved, create ops ticket to have them update their rules.

**Actual Behavior**
Alerts not produced for critical errors

**Expected Behavior**
Alerts produced for critical errors

**Environment:**
- Version: v0.4.1

**Additional Context**
